App/ProtoControllers/HautResponsable/Utilisateur.php

```diff
@@ -737,20 +736,18 @@ private static function updateSoldeUtilisateur($data, \includes\SQL $sql)
         $returnStd = $sql->query($req);
 
         $returnExc = true;
-        if ($config->isCongesExceptionnelsActive()) {
-            $typeAbsencesExceptionnels = \App\ProtoControllers\Conge::getTypesAbsences($sql, 'conges_exceptionnels');
-            if(empty($typeAbsencesExceptionnels)){
-                return $returnStd;
-            }
-            foreach ($typeAbsencesExceptionnels as $typeId => $info) {
-                $valuesExc[] = '(0, \''
-                                . $data['soldes'][$typeId] . '\', 0, "'
-                                . $data['oldLogin'] . '", '
-                                . (int) $typeId . ')';
-            }
-            $req = 'REPLACE INTO conges_solde_user (su_nb_an, su_solde, su_reliquat, su_login, su_abs_id) VALUES ' . implode(",", $valuesExc);
-            $returnExc = $sql->query($req);
-        }
+        $typeAbsencesExceptionnels = \App\ProtoControllers\Conge::getTypesAbsences($sql, 'conges_exceptionnels');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\App\ProtoControllers\Conge' in method 'getFormListeUsers'.](https://app.codacy.com/manual/Libertempo/web/pullRequest?prid=5122180)